contrib/exporters/secadvisor/mod/transform.go

```diff
@@ -38,7 +38,12 @@ func NewTransform(cfg *viper.Viper) (interface{}, error) {
 		return nil, err
 	}
 
-	resolver, err = NewResolveMulti(resolver.(Resolver))
+	resolver2, err := NewResolveVm(cfg)
```

Comment:
I guess you are missing a file - as I don't see the implementation of `NewResolveVM` anywhere ... also if resolution is a specific kind of Vm such as `Xen` then rather call the handler `NewResolveXen`

Reply:
You are right. I forgot to add the file resolve_vm.go.